packages/manager/modules/pci/src/projects/project/notebooks/components/configuration-command/configuration-command.html

```diff
@@ -0,0 +1,11 @@
+<p data-translate="pci_notebook_configuration_command_description"></p>
+<pre class="bg-dark">
+    <oui-spinner data-ng-if="$ctrl.loading" data-size="s"></oui-spinner>
```

Comment:
Maybe a `oui-skeleton` would be more adequate here ?

Reply:
The background of the element is dark here. hence a `oui-skeleton` would not be visible I guess...

Reply:
I'm not sure the contrast of the `oui-spinner` with a dark background color either

Reply:
ok... Thats right. Using `oui-skeleton` now :)